src/core/varlink.c

```diff
@@ -680,6 +701,35 @@ static int manager_varlink_init_user(Manager *m) {
         if (MANAGER_IS_TEST_RUN(m))
                 return 0;
 
+        r = manager_setup_varlink_server(m);
+        if (r < 0)
+                return log_error_errno(r, "Failed to set up varlink server: %m");
+        bool fresh = r > 0;
+
+        FOREACH_STRING(a,
+                       "systemd/io.systemd.Manager") {
```

Comment:
why FOREACH_STRING() if there's only a single string

Reply:
actually, I'd like to keep it. It will be necessary in follow up patches, and also symmetric with `manager_varlink_init_system()`